### ISSUE TITLE
[accessibility] Activity graph does not have an accessible text version

### ISSUE THREAD

**Issue by sandbergja** [metacpan/metacpan-web#3108]
**Describe the bug** A clear and concise description of what the bug is.

**To Reproduce** Steps to reproduce the behavior:

1. In Firefox, go to https://metacpan.org/pod/Test2
2. There is a graph showing activity on Test2 in the past 24 months.  Right click it.
3. Select "Inspect Accessibility Properties"
4. This will open a tree view with the element type and the accessible name. Note that the accessible name for the image is an empty string.

**Expected behavior**

The SVG image has an alternative, so that screen reader users can also know what the activity has been like on a distribution.

**Desktop (please complete the following information):**

- OS: Windows
- Browser Firefox
- Version 127

**Comment by oalders** [comment]
Closed via #3332
